composer.json

```diff
@@ -18,7 +18,8 @@
         "friendsofphp/php-cs-fixer": "^3.0",
         "phpunit/phpunit": "^9.0",
         "phpstan/phpstan": "^0.12.18",
-        "vimeo/psalm": "^4.0"
+        "vimeo/psalm": "^4.0",
+        "swiftmailer/swiftmailer": "^6.3"
```

Comment:
[Swiftmailer is deprecated](https://github.com/swiftmailer/swiftmailer). We should use [Symfony Mailer](https://symfony.com/doc/current/mailer.html) instead.